js/jquery.webform-ajax-slide.js

```diff
@@ -70,6 +70,12 @@
       // Set dummy dimensions to current container dimensions.
       $loadingdummy.css({height: $slide.height() + 'px', width: $slide.width() + 'px'});
 
+      // Fetch payment status messages if they exist.
+      $loadingdummy.find('.payment-success').remove();
+      ajax.form.find('.payment-success').each(function () {
+        $('<p class="payment-success">').html($(this).html()).appendTo($loadingdummy);
+      });
```

Comment:
Although `<p>` is the semantically correct markup, it does not work best with our default styles :grimacing: If we want to go the lazy way, perhaps we should use `<div class="payment-success form-label">` instead? :thinking:

Reply:
I’m happy to follow your suggestion. I’d just use whatever works quickest in this case.